<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DangKiVe</class>
 <widget class="QWidget" name="DangKiVe">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1029</width>
    <height>590</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="tblLichSuVaoRa">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>340</y>
     <width>621</width>
     <height>211</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Mã Lịch Sử</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Biển số</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Thời Gian Vào</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Thời Gian Ra</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Phí</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Vé</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Thời gian vào</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>100</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Thời gian ra</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtBienSo">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>30</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Biển Số</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnXeVao">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>140</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Xe vào</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnLamMoi">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>240</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Làm mới</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnXeRa">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>140</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Xe ra</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtMaVaoRa">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>30</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Mã Lịch Sử</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbLoaiVe">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>100</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnThoat">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Thoát</string>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="dtThoiGianVao">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>100</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="dtThoiGianRa">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>100</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtTuKhoa">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>240</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Tìm Kiếm Xe Ra, Vào</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnTimKiem">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>240</y>
     <width>93</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Tìm Kiếm</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
